src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -162,6 +162,9 @@
 			<x:Double x:Key="MaterialOutlinedTextBoxMinHeight">56</x:Double>
 			<Thickness x:Key="MaterialOutlinedTextBoxBorderPadding">1</Thickness>
 			<x:Double x:Key="MaterialOutlinedTextBoxBorderThicknessFocused">2</x:Double>
+			<Thickness x:Key="MaterialOutlinedTextBoxPaddingFocused">0</Thickness>
+			<x:Double x:Key="MaterialOutlinedTextBoxBorderThicknessPointerOver">2</x:Double>
+			<Thickness x:Key="MaterialOutlinedTextBoxPaddingPointerOver">0</Thickness>
 			<!--#endregion-->
 		</ResourceDictionary>
 
```

Comment:
For MaterialFilledTextBoxStyle we should do the same for :
https://github.com/unoplatform/Uno.Themes/blob/c35a540d0dfe557519603a4fad3b38e1c72ba7df/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L375